quipucords/api/migrations/0028_migrate_rawfact_value_to_jsonfield.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.17 on 2023-03-21 08:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0027_migrate_search_directories_to_jsonfield"),
```

Comment:
could you edit the migration dependency in the first commit?
(currently 058be18a18e0a26946539d3c6db61502b3f18a75)

Reply:
squashing works too =D